<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TritKi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="article_list_layout">
        <item>
         <layout class="QHBoxLayout" name="search_layout">
          <item>
           <widget class="QLineEdit" name="search_text">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search_button">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="article_list">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="article_button_layout">
          <item>
           <spacer name="article_button_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="delete_article">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="new_article">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="article_viewport">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="read_tab">
        <attribute name="title">
         <string>Read</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTextBrowser" name="page_view">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="openLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="edit_tab">
        <attribute name="title">
         <string>Edit</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="edit_title_layout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Title </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_title"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="edit_format_layout">
           <item>
            <widget class="QToolButton" name="edit_bold">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="edit_italic">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="tool_separator_1">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="edit_quote">
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="edit_heading"/>
           </item>
           <item>
            <widget class="Line" name="tool_separator_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="edit_wikilink">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 255);</string>
             </property>
             <property name="text">
              <string>[[Page]]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="edit_link">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 255);</string>
             </property>
             <property name="text">
              <string>[Text](url)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="tool_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="edit_save">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="SpellTextEdit" name="page_edit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="history_tab">
        <attribute name="title">
         <string>History</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="page_history"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpellTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">tritki.gui.spelltextedit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
